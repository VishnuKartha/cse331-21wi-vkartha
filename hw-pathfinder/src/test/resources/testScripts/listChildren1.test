# Creates a single graph and adds 2 nodes(n1,n2) to it and an edge from n1 to n2.
# Then lists the children of of n1 in graph1 and n2 in graph1(no children for n2)

# Create a Graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge between the two nodes
AddEdge graph1 n1 n2 10.000

#List the children of n1 and n2
ListChildren graph1 n1
ListChildren graph1 n2