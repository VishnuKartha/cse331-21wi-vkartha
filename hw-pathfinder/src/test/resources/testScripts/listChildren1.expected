# Creates a single graph and adds 2 nodes(n1,n2) to it and an edge from n1 to n2.
# Then lists the children of of n1 in graph1 and n2 in graph1(no children for n2)

# Create a Graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add an edge between the two nodes
added edge 10.000 from n1 to n2 in graph1

#List the children of n1 and n2
the children of n1 in graph1 are: n2(10.0)
the children of n2 in graph1 are: